### DISCUSSION TITLE
sitemap2 插件的 hostname 选项应当提供示例

### DISCUSSION THREAD

**Discussion by guyutongxue** [vuepress-theme-hope/vuepress-theme-hope#1990] [Q&A] [answered]
这里的 `hostname` 选项期望的格式是 `https://example.com` 而不是 `example.com`。不带 `https://` 或 `http://` 的话，构建时会产生没有详细原因的“URL 格式错误”错误。

但这个字段的[文档](https://vuepress-theme-hope.github.io/v2/sitemap/config.html#plugin-options)说要填入“域名”，我认为这个用词不太准确。此外，BOM 的 `window.location.hostname` 也是 `example.com`，`hostname` 这个字段名也容易引起误会。

我建议在文档中增加示例，比如 `Eg: https://example.com` 之类的？或者在后面处理的时候自动加上协议名？

**Comment by Mister-Hope** [comment]
很不错的建议。

**Comment by Mister-Hope** [comment, marked as answer]
Improved in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/77983610b97e66180c2dc7033ad1786df2c208da
